src/lib/logical_query_plan/export_node.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <string>
+
+#include "base_non_query_node.hpp"
+#include "enable_make_for_lqp_node.hpp"
+#include "import_export/file_type.hpp"
+#include "storage/table_column_definition.hpp"
+
+namespace opossum {
+
+/**
+ * This node type represents the COPY TO management command.
+ */
```

Comment:
The ImportNode says, `IMPORT / COPY FROM management command.` Should we say `EXPORT /...` here?

Reply:
The SQLParser supports both the old IMPORT FROM ... INTO and COPY ... FROM ... and generates an ImportStatement from both, while it only makes an ExportStatement from COPY ... TO ...
That's why we have a difference here.

Reply:
Alright